ZenPacks/zenoss/OpenStackInfrastructure/tests/test_hostmap.py

```diff
@@ -49,7 +49,10 @@ def resolve_names(names):
                 'test2': '1.2.3.5',
                 'test2.example.com': '1.2.3.5',
                 'test1.localdomain': '127.0.0.1',
-                'test2.localdomain': '127.0.0.1'
+                'test2.localdomain': '127.0.0.1',
+
```

Comment:
It's two different sets of names used by different unit tests, so i left a space to separate them visually